tests/test_relation_u.py

```diff
@@ -50,7 +50,13 @@ def test_invalid_join():
         rel = dj.U('language') * dict(language="English")
 
     def test_aggregations(self):
```

Comment:
In `test_relational_operand.py`, under `TestRelational.test_aggregate()`, could you add one or more tests that use `aggr` in place of `aggregate`? It would be easiest for you to add this as the PR is still open.